src/generators/testing/TestingGenerator.res

```diff
@@ -110,14 +110,153 @@ let drawGrid = () => {
   }
 }
 
-let drawSteveBodyCuboid = (x, y, scale, direction, center) => {
+let isAligned = ({rectangle, _}: Minecraft.Cuboid.Face.t) => {
```

Comment:
@NinjolasNJM could you also add comments on what's happening in this file as well. E.g. it looks like `isAligned` checks if the `x` and `y` coordinates of a face are both even numbers? It would be good to explain how that relates to "alignment". Same idea with the rest of this code - add as many comments as possible to explain what's happening.